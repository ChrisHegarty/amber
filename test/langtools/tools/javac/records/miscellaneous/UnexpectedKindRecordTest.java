/*
 * @test /nodynamiccopyright/
 * @summary check that record is not used as a type name
 * @compile/fail/ref=UnexpectedKindRecordTest.out -XDrawDiagnostics UnexpectedKindRecordTest.java
 */

class UnexpectedKindRecordTest {

    record R(int i) {
        public int i() { return i; }
        public int i() { return i; }
    }
}
