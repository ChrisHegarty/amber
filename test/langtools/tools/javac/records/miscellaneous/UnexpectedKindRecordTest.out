UnexpectedKindRecordTest.java:11:20: compiler.err.already.defined: kindname.method, i(), kindname.class, UnexpectedKindRecordTest.R
1 error
